django_diplom/cart/views.py

```diff
@@ -0,0 +1,199 @@
+from typing import Optional, Any, Dict
+
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.core.mail import send_mail
+from django.db.models import Count
+from django.http import HttpResponse
+from django.shortcuts import render, get_object_or_404, redirect
+from django.urls import reverse
+from django.views import View
+from django.views.generic import TemplateView
+
+from catalog.models import Product, Addon, Category
+from cart.models import CartProduct, CartAddon, Customer, Cart
+from django_diplom.settings import EMAIL_HOST_USER
+
+from .forms import OrderForm
+
+# add_to_cart/hamurger/?is_addon=0
+from .mixins import CartMixin
+from .utils import recalc_cart
+
+
+def add_to_cart(request, product_slug):
+    slug_product = Product.objects.get(slug=product_slug)
```

Comment:
```suggestion
    product = Product.objects.get(slug=product_slug)
```

Это же просто `product` :wink: